msal4j-brokers/src/main/java/com/microsoft/aad/msal4jbrokers/MsalRuntimeBroker.java

```diff
@@ -176,4 +176,37 @@ public boolean isBrokerAvailable() {
             return false;
         }
     }
+
+    /**
+     * Toggles whether or not detailed MSALRuntime logs will appear in MSAL Java's normal logging framework.
+     *
+     * If enabled, you will see logs directly from MSALRuntime, containing verbose information relating to telemetry, API calls,successful/failed requests, and more.
+     * These logs will appear alongside MSAL Java's logs (with a message indicating they came from MSALRuntime), and will follow the same log level as MSAL Java's logs (info/debug/error/etc.).
+     *
+     * If disabled (default), MSAL Java will still produce some logs related to MSALRuntime, particularly in error messages, but will be much less verbose.
+     *
+     * @param enableLogging true to enable MSALRuntime logs, false to disable it
+     */
+    public void toggleLogging(boolean enableLogging) {
```

Comment:
nit - rename the method to enableLog or enableBrokerLog. toggleLogging seems ambiguous. toggleLogging - false -> does this mean do not toggle? toggleLogging - true -> does this mean toggle the current value?